lib/gui/os/window-progress.js

```diff
@@ -18,7 +18,14 @@
 
 const electron = require('electron');
 const utils = require('../../shared/utils');
-const packageJSON = require('../../../package.json');
+const progressStatus = require('../models/progress-status');
+
+/**
+ * @summary The title of the main window upon program launch
+ * @type {String}
+ * @protected
+ */
+const entryTitle = document.title;
```

Comment:
Minor nit, but `entryTitle` doesn't sound right to me. Perhaps `initialTitle` or `defaultTitle` instead?